src/actions/index.ts

```diff
@@ -329,3 +329,24 @@ export const sendInvite: AsyncActionCreator = (inviteEmail: string) => {
     inviteEmail
   );
 };
+
+export const sendAppInstallText: AsyncActionCreator = (
+  mobileNumber: string
+) => {
+  return wrapApiCallAction(
+    async (dispatch, getState) => {
+      await callApi<SendAppInstallTextApiEndpoint>(
+        {
+          endpoint: ApiEndpoint.SEND_APP_INSTALL_TEXT,
+          params: undefined,
```

Comment:
Do you have to explicitly set this?

Reply:
tried removing it and typescript complained -- i guess the property is required, even though it can be undefined?